app/views/admin/events/_event_menu.html.erb

```diff
@@ -6,5 +6,6 @@
     <li><%= link_to_active "Waiting list e-mail", edit_admin_event_template_path(@event, :waiting_list_mail) %></li>
     <li><%= link_to_active "Rejection e-mail", edit_admin_event_template_path(@event, :rejection_mail) %></li>
     <li><%= link_to_active "Reminder e-mail", edit_admin_event_template_path(@event, :reminder_mail) %></li>
+    <li><%= link_to_active "Reminder for attendance e-mail", edit_admin_event_template_path(@event, :reminder_attendance_mail) %></li>
```

Comment:
Should we call it "Confirmation reminder e-mail" since this is to remind applicants to click the confirmation link?

Reply:
Sure!